### DISCUSSION TITLE
小吃店也該導入AI嗎？這個原則教你判斷

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#718] [General]
https://futurecity.cw.com.tw/article/891
新時代的科技引領舊時代的產物

**Comment by Valentinaa0214** [comment]
我覺得 不要為了AI而AI 這句話講得很有道理，因為現今太多行業開始引入與利用人工智慧提升他們的效率與效益，但我認為不是所有行業都需要把重心放在上面的，要先從企業的關鍵問題著手。

**Comment by HsuChen88** [comment]
對於某些小型生意來說，我認為AI能幫忙的沒有想像中那麼多，他們需要的或許只是一個將流程自動化的系統而已。

**Comment by yjpxi** [comment]
我覺得可以評估導入AI的成本是否能夠被提升的效率和增強的顧客體驗所抵消。如果投資回報率高，則導入AI是值得的。
